koku/api/ingress/reports/view.py

```diff
@@ -86,6 +86,7 @@ def post(self, request):
             "reports_list": request.data.get("reports_list"),
             "bill_year": request.data.get("bill_year"),
             "bill_month": request.data.get("bill_month"),
+            "customer": request.user.customer_id,
```

Comment:
Should we log the data object here for debugability? Should we get the schema since the customer id isn't a super visible value?

Reply:
So I was looking at that but the silly models wouldnt let me :frowning_face: I was probably just over thinking/doing something wrong. Happy to change it if I can figure it out.

Reply:
I don't see any logging, but I guess we will be able to use gabi to make sure the schema value from the request makes sense.